src/background/index.ts

```diff
@@ -18,10 +18,21 @@ const WATCH = __WATCH__;
 
 if (WATCH) {
     const PORT = __PORT__;
+    const ALARM_NAME = 'socket-close';
+    const PING_INTERVAL_IN_MINUTES = 1 / 60;
```

Comment:
Hmmm, what is a possible outcome of such short intervals? Will it make the page persistent?

Reply:
Yes, short intervals can be wasteful (including staying in memory for prolonged periods of time). Chrome acttually throttles periodic alarms to once a minute and tries to limit non-persistent background lifespan to 5 minutes.

However, in this case performance is not an issue because this code runs only on debug watch build and only when there is no active socket connection.